internal/app/recordtester/recordtester_app.go

```diff
@@ -341,6 +302,111 @@ func (rt *recordTester) Start(fileName string, testDuration, pauseDuration time.
 	// <-downloader.Done()
 	// glog.Infof(`Pulling stopped after %s`, time.Since(started))
 	// exit(55, fileName, *fileArg, err)
+
+	glog.Info("Record testing done, waiting for Asset to be available")
+	if err = rt.isCancelled(); err != nil {
+		return 0, err
+	}
+
+	assetId := ""
+	wait_interval := 30 * time.Second
+	elapsed_time := 0 * time.Second
+asset:
+	for {
+		glog.Infof("Waiting %s for asset, elapsed=%s", wait_interval, elapsed_time)
+		time.Sleep(wait_interval)
+		elapsed_time = elapsed_time + wait_interval
+		if err = rt.isCancelled(); err != nil {
+			return 0, err
+		}
+
+		assets, err := rt.lapi.ListAssets() // assets are retrieved in order of most recent first
+		if err != nil {
+			glog.Errorf("Error listing assets err=%v", err)
+			return 241, err
+			// exit(241, fileName, *fileArg, err)
+		}
+		for _, asset := range *assets { // TODO: only need to check first n assets due to ordering
+			if asset.Name == fmt.Sprintf("live-to-vod-%s", sess.ID) {
+				assetId = asset.ID
+				break asset
+			}
+		}
+
+		// TODO: timeout required here?
+	}
+
+	task_name := fmt.Sprintf("live-to-vod-transcode-%s", sess.ID)
+	createTranscodeTaskResp, _ := rt.lapi.CreateTranscodeTask(assetId, task_name, livepeerAPI.StandardProfiles[0])
+	if err != nil {
```

Comment:
You're checking the error here but you're ignoring it in the line above. 
```suggestion
	createTranscodeTaskResp, err := rt.lapi.CreateTranscodeTask(assetId, task_name, livepeerAPI.StandardProfiles[0])
	if err != nil {
```